src/clients/go/samples/basic/go.mod

```diff
@@ -2,4 +2,6 @@ module basic
 
 go 1.17
 
-require github.com/tigerbeetle/tigerbeetle-go v0.13.78
+require github.com/tigerbeetle/tigerbeetle-go v0.0.0
+
+replace github.com/tigerbeetle/tigerbeetle-go => ../../
```

Comment:
Same here, remove mentioning of the version, use the code from the repo instead.